src/doc/ndarray_for_numpy_users/mod.rs

```diff
@@ -524,6 +524,30 @@
 //! </td></tr>
 //! </table>
 //!
+//! ## Type conversions
+//!
+//! In `ndarray`, conversions between datatypes are done with `mapv()` by
+//! passing a closure to convert every element independently.
+//! For the conversion itself, we have several options:
+//! - `std::convert::From` ensures perfect, safe conversions at compile-time and
+//!   is generally recommended.
+//! - `std::convert::TryFrom` can be used for potentially unsafe conversions. It
+//!   will panic if values are encountered at runtime that cannot be converted
+//!   perfectly.
+//! - The `as` keyword performs a *saturating* cast since Rust 1.45 and should
+//!   only be used for safe, clear conversions such as upcasting.
```

Comment:
The note about saturating casts is for floats only, not for all `as` conversions. We need a doc link anyway to explain the full rules, I think.

```suggestion
//! - The `as` keyword performs a cast which can convert between primitive types. [Other documentation links here]
```

Reply:
That is a great suggestion, thank you!